tests/acceptance/PreventSiteAccessCept.php

```diff
@@ -0,0 +1,63 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected vipgoci-phpcs-scan-obaepm.php, but found vipgoci-phpcs-scan-OBaEpm.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).